x-pack/plugins/apm/public/utils/formatters.ts

```diff
@@ -141,3 +142,32 @@ export function asPercent(
   const decimal = numerator / denominator;
   return numeral(decimal).format('0.0%');
 }
+
+export function roundToPlaces(value: number, places: number) {
+  const multiplier = 10 * places;
+  return Math.floor(value * multiplier) / multiplier;
+}
```

Comment:
Maybe just use `numeral` for this `numeral(value).format('00.[00]')` (we want 2 decimal places everywhere anyway)

Reply:
I just used `asDecimal` for now because it was there but you all can tweak it if 2 places makes more sense overall.